auth-web/src/components/auth/AccountLoginOptionPicker.vue

```diff
@@ -1,30 +1,29 @@
 <template>
   <v-row>
     <v-col
+      sm="12"
+      md="6"
       class="d-flex align-stretch"
-      sm="12" md="6"
       v-for="authOption in authOptions"
       :key="authOption.type"
     >
       <v-card
+        class="account-card elevation-2 pa-10 pt-9 d-flex flex-column text-center"
+        :class="{ 'active': authType === authOption.type }"
         flat
-        outlined
         hover
-        elevation="1"
-        class="account-card text-center pa-7 d-flex flex-column"
         @click="selectAuthType(authOption.type)"
-        :class="{ 'active': authType === authOption.type }"
       >
         <div class="account-type__icon mb-6 mt-2">
           <v-icon color="grey">{{authOption.icon}}</v-icon>
```

Comment:
The BCeID logo will be missing here.
I have used the 2FA icon as a substitute, are we planning to keeping that?

Reply:
Fine for now - I'll get the BCeID asset in there in my next PR